instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import CommentSection from "../CommentSection/CommentSection";
+
+class PostContainer extends Component {
+  render() {
+    return (
+         //doesn't matter if you map in app or here - style, baby!
+      //mapping the data AND formatting it at the same time.  Just wild stuff
+      //map(post - > post becomes the starter for everythign!!
+      // map over everything, return "post" for each object - so we can
+      // just pull off surface level info right here!
+      // It's so BONKERS how much common sense this is!
+
+      //this.props works because constructore is auto included! ES6 goodness
+      <div className="post-container">
+        {this.props.filteredPosts.length === 0
+          ? this.props.data.map(post => {
```

Comment:
Very clever  use of the ternary operator here to contionally render one of 2 components either a filtered list by the search, or all of the posts. This is great! I would break these into sub components tho like a `filteredList` and `allPost` component and the ternary is used to select which one is rendered instead of writing the JSX here.